test/gst-msdk/util.py

```diff
@@ -35,6 +35,7 @@ def get_supported_format_map():
     "422H"  : ("y42b", "Y42B"),
     "444P"  : ("y444", "Y444"),
     "P010"  : ("p010-10le", "P010_10LE"),
+    "P012"  : ("p012le", "P012_LE"),
```

Comment:
`p012le`  should be `p012-le` (as seen in `gst-inspect rawvideoparse` format property)

Reply:
Ok, let me to update soon, thanks
(92): p012-le          - GST_VIDEO_FORMAT_P012_LE